data/text/common_1.asm

```diff
@@ -1155,7 +1155,7 @@ Text_BreedingIsNotPossible::
 UnknownText_0x1bd0d8::
 	text "The compatibility"
 	line "is @"
-	deciram wd265, 1, 3
+	deciram wDeciramBuffer, 1, 3
```

Comment:
I'd probably use `wBreedmonCompatibility` here, given that it is already defined anyway